src/language/__tests__/schema-kitchen-sink.graphql

```diff
@@ -17,6 +17,7 @@ type Foo implements Bar {
   four(argument: String = "string"): String
   five(argument: [String] = ["string", "string"]): String
   six(argument: InputType = {key: "value"}): Type
+  seven(argument: Int = null): Type
```

Comment:
Right, so by setting the default value of an input argument to `null` you're making it impossible to know if the user passed in `null` vs the user omitted the input value entirely.

```graphql
six(argument: Int): Type
seven(argument: Int = null): Type
```

In `six`, the value of `argument` could be `null | Int | undefined`.

In `seven`, the value of `argument` could be `null | Int`.

Since both `null` and `undefined` are falsy, I can't think of a case where one would explicitly set the **default** value to `null` in order to omit the `undefined` case.

Maybe when using Flow/TypeScript removing this 3rd case makes it easier to understand the code?